openmsupply/open-msupply
# 1220 integrate new name fields

Fixes #1220

# 👩🏻‍💻 What does this PR do? 
- Integrate new om_gender and om_created_datetime to name translation

# 🧪 How has/should this change been tested? 
- Create patient in central
- Sync
- Go to `Patient` and see patient 
OR 
- Create patient in om
- Sync
- Delete db
- Reinitialise

## 💌 Any notes for the reviewer?
